examples/bert/bert_cuda_gpu.json

```diff
@@ -118,7 +118,6 @@
         "perf_tuning": {
             "type": "OrtPerfTuning",
             "config": {
-                "enable_cuda_graph": true,
```

Comment:
Will this speed up the test?

Reply:
This is not for test speed up. turns out the onnx model here is not compatible with cuda graph so perf tuning only returned the input model as best model. so I disabled it so that the perf tuning pass here actually finds something useful.